oci/table_oci_core_subnet.go

```diff
@@ -0,0 +1,291 @@
+			{
+				Name:        "display_name",
+				Description: "A user-friendly name. Does not have to be unique, and it's changeable.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The subnet's Oracle ID (OCID).",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "lifecycle_state",
+				Description: "The subnet's current state.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "The date and time the subnet was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+			{
+				Name:        "availability_domain",
+				Description: "The subnet's availability domain.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "cidr_block",
+				Description: "The subnet's CIDR block.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "dhcp_options_id",
+				Description: "The OCID of the set of DHCP options that the subnet uses.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "dns_label",
+				Description: "A DNS label for the subnet, used in conjunction with the VNIC's hostname and VCN's DNS label to form a fully qualified domain name (FQDN) for each VNIC within this subnet.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "ipv6_cidr_block",
+				Description: "For an IPv6-enabled subnet, this is the IPv6 CIDR block for the subnet's private IP address space.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Ipv6CidrBlock"),
+			},
+			{
+				Name:        "ipv6_public_cidr_block",
+				Description: "For an IPv6-enabled subnet, this is the IPv6 CIDR block for the subnet's public IP address space.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Ipv6PublicCidrBlock"),
+			},
+			{
+				Name:        "ipv6_virtual_router_ip",
+				Description: "For an IPv6-enabled subnet, this is the IPv6 address of the virtual router.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Ipv6VirtualRouterIp"),
+			},
+			{
+				Name:        "prohibit_public_ip_on_vnic",
+				Description: "Indicates whether VNICs within this subnet can have public IP addresses.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "route_table_id",
+				Description: "The OCID of the route table that the subnet uses.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "subnet_domain_name",
+				Description: "The subnet's domain name, which consists of the subnet's DNS label.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "vcn_id",
```

Comment:
Ordering of column is not correct ,`vcn_id` and `route_table_id` are important columns and should be at top.